jenkins/Dockerfile.integration.centos7

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This change should not have been made.  The file originated in 2019 and continued to be updated in 2020.

Reply:
ah I see. thx